app/controllers/users/passwords_controller.rb

```diff
@@ -27,26 +28,10 @@ def user_params
     end
 
     def handle_success
-      re_encrypt_active_profile
-
       bypass_sign_in current_user
 
+      flash[:recovery_code] = @update_user_password.recovery_code
```

Comment:
Ah. I see where the confusion is. By renaming it to `recovery_code`, I made it clearer. What `active_profile.encrypt_pii` returns is the recovery code, which is confusing, and something I've been meaning to refactor.